### PR TITLE
Add rights data to transfer METS

### PR DESCRIPTION
**Pull request by cole**
Connects to https://github.com/archivematica/Issues/issues/587

### CODE REVIEW

File: src/MCPClient/lib/clientScripts/create_transfer_mets.py [hunk 1]
```diff
@@ -376,6 +601,74 @@ def event_to_premis(event):
...
 
 
+def rights_to_premis(rights, file_uuid):
+    """
+    Converts an RightsStatement model to a PREMIS event object via metsrw.
+
+    Returns:
+        lxml.etree._Element
+    """
+    VALID_RIGHTS_BASES = ["copyright", "institutional policy", "license", "statute"]
```
Comment (cole) [inline src/MCPClient/lib/clientScripts/create_transfer_mets.py:611]: I'd prefer to move this up to module level, but` ensure_no_mutable_globals` complains (even if I make this an _immutable_ `frozenset`, lol). I don't think thats a valid complaint, but I'll submit an issue for that.

File: src/MCPClient/lib/clientScripts/create_transfer_mets.py [hunk 2]
```diff
@@ -197,6 +208,33 @@ def lookup_file_data(relative_path, db_base_path, lookup_kwargs):
     return file_obj
 
 
+def lookup_file_rights(file_uuid, transfer_uuid):
+    TRANSFER_RIGHTS_LOOKUP_UUID = "45696327-44c5-4e78-849b-e027a189bf4d"
+    FILE_RIGHTS_LOOKUP_UUID = "7f04d9d4-92c2-44a5-93dc-b7bfdf0c1f17"
```
Comment (cole) [inline src/MCPClient/lib/clientScripts/create_transfer_mets.py:213]: I'd prefer not to use hardcoded UUIDs here, but that's how the model is. In the meantime, I'd prefer to move this up to module level, but` ensure_no_mutable_globals` complains.

**Review by sevein — COMMENTED** [review]
I've found some validation issues - via http://pim.fcla.edu.
Full report here: https://pastebin.com/ZWfmFAE4.

* Invalid content was found starting with element 'premis:eventDetail'.

  I think that in PREMIS 3 the element has been renamed.

* The type definition cannot be abstract for element premis:object.

  ```xml
  <mets:techMD ID="techMD_1" CREATED="2019-03-26T18:02:02" STATUS="current">
    <mets:mdWrap MDTYPE="PREMIS:OBJECT">
      <mets:xmlData>
        <premis:object
          xmlns:premis="http://www.loc.gov/premis/v3"
          xsi:schemaLocation="http://www.loc.gov/premis/v3
          http://www.loc.gov/standards/premis/v3/premis.xsd"
          version="3.0">
          [...]
  ```

  Missing `type`?

* Element `premis:object` must have no character or element information item [children], because the type's content type is empty.
  
  I think this means that `xsi:type="file"` is missing!

  ```xml
  <premis:object xmlns:premis="http://www.loc.gov/premis/v3" xsi:schemaLocation="http://www.loc.gov/premis/v3 http://www.loc.gov/standards/premis/v3/premis.xsd" version="3.0">
    [...]
  </premis:object>
  ```

* Element 'premis:objectCharacteristicsExtension' cannot have character [children], because the type's content type is element-only.
  
  There's somethign wrong with the XML we're creating, see the `object_characteristics_extension` block - shouldn't be there.
  
  ```xml
  <premis:objectCharacteristicsExtension>
    <ffprobe>
    </ffprobe>object_characteristics_extension...
  <premis:objectCharacteristicsExtension>
  ```

* Attribute 'version' is not allowed to appear in element 'premis:object'.<br />
  Attribute 'version' is not allowed to appear in element 'premis:rightsStatement'.

  The error is clear, but why?

**Comment by sevein** [comment]
I'll be addressing the validation issues in a different pull request.
The code is working, the new transfer METS is generated and it includes rights!

**Review by sevein — APPROVED** [review]

**Pull request merged**